script/generate-regex.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
+const path = require('path')
+
+const regenerate = require('regenerate')
+
+const set = regenerate()
+  .add(require('unicode-12.1.0/Sequence_Property/Basic_Emoji/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_Flag_Sequence/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_Modifier_Sequence/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_Tag_Sequence/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_ZWJ_Sequence/index.js'))
```

Comment:
```suggestion
  .add(require('unicode-12.1.0/Sequence_Property/Emoji_ZWJ_Sequence/index.js'))
  .add(require('unicode-12.1.0/Block/Halfwidth_And_Fullwidth_Forms/code-points.js'))
```
I ran into this today, would you consider adding the Halfwidth and Fullwidth Forms block tp tjos PR?

Here's a link to the heading at issue: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ja.md#型定義ファイルとは何ですか-またどのように入手できますか

Here's the current github-slugger result vs. the correct slug:
```Diff
-型定義ファイルとは何ですか？-またどのように入手できますか？
+型定義ファイルとは何ですか-またどのように入手できますか
```

I confirmed that this change (adding the Halfwidth and Fullwidth Forms block) fixes the slug for this heading.